src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/markup/Head.java

```diff
@@ -60,6 +60,7 @@
     private boolean showTimestamp;
     private DocPath mainStylesheet;
     private List<DocPath> additionalStylesheets = Collections.emptyList();
+    private List<DocPath> additionalScripts = Collections.emptyList();;
```

Comment:
Suggest moving this down a few lines, after `mainBodyScript` and `scripts`.

Reply:
While I realize the maybe-useful similarity between `additionalStylesheets` and `additionalScripts`, the similarity between `scripts` and `additionalScripts` leaves me asking, "why both?". I'm  not sure I would go as far as a systemic rename of `additionalScripts` to `userScripts`, but maybe a comment on the declaration would be helpful to future readers.

Reply:
Moved declaration down and added a comment.